pkg/filter/csrf/csrf.go

```diff
@@ -90,16 +88,16 @@ func (f *Filter) Decode(ctx *http.HttpContext) filter.FilterStatus {
 	salt := ctx.Request.Header.Get(csrfSalt)
 
 	if salt == "" {
-		bt, _ := json.Marshal(http.ErrResponse{Message: f.cfg.ErrorMsg})
-		ctx.SendLocalReply(stdHttp.StatusForbidden, bt)
+		errResp := http.Forbidden.WithError(fmt.Errorf(f.cfg.ErrorMsg))
```

Comment:
if the format func is not used when creating a new err, use `errors.New` instead of `fmt.Errorf`. `fmt.Errorf` only used when need format
```suggestion
		errResp := http.Forbidden.WithError(errors.New(f.cfg.ErrorMsg))
```